README.md

```diff
@@ -84,6 +84,8 @@ Returns a new focus trap on `element` (one or more "containers" of tabbable node
 
 - **onActivate** {function}: A function that will be called when the focus trap activates.
```

Comment:
Let's update this doc to state that `onActivate()` is called _before_ the initial focus node gets focused.

Reply:
Not quite true though unless we make a potentially breaking change by moving `onActivate()` before `updateTabbableNodes()`